roles/pulp/tasks/install.yml

```diff
@@ -49,20 +55,38 @@
         system: true
       when: developer_user is not defined
 
+    - name: Add user {{ pulp_user }} to {{ pulp_group }} group
+      user:
+        name: '{{ pulp_user }}'
+        groups:
+          - '{{ pulp_group }}'
+        append: true
+
+    - name: Add user {{ developer_user }} to {{ pulp_group }} group
+      user:
+        name: '{{ developer_user }}'
+        groups:
+          - '{{ pulp_group }}'
+        append: true
+      when: developer_user is defined
+
+    - name: Reset ssh conn to allow user changes to affect when ssh user and pulp user are the same
```

Comment:
Should this have a `when` to detect this?

Reply:
I tried that, but got a warning saying that it is not allowed with the 'reset_connection' metatask.